hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCreateSuite.java

```diff
@@ -108,7 +107,7 @@
 import org.apache.tuweni.bytes.Bytes32;
 import org.junit.jupiter.api.Tag;
 
-@HapiTestSuite
+// @HapiTestSuite
```

Comment:
Q: SHould this be reverted?
```suggestion
 @HapiTestSuite
```

Reply:
Fixed